image/radon/example.py

```diff
@@ -0,0 +1,20 @@
+import tensorflow as tf
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.